container/root/etc/services-available/php-fpm/run

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/with-contenv bash
+
+exec sudo -u www-data php5-fpm -F -O 2>&1 | sed -u 's,.*: \"\(.*\)$,\1,'| sed -u 's,"$,,' 1>&1
```

Comment:
the piping works, but the signal passing does not. if the container is being taken down, nginx will respond to the draining, which will close off the backends. in this case, we're working around the problem, but its not different than it is currently today